frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -69,6 +88,54 @@ const cancelUpdate = (cv: ClusterVersionKind) => {
 
 export const clusterAutoscalerReference = referenceForModel(ClusterAutoscalerModel);
 
+const getMachineConfigPoolByName = (
+  machineConfigPools: MachineConfigPoolKind[],
+  name: string,
+): MachineConfigPoolKind => {
+  return machineConfigPools?.find((mcp) => mcp.metadata.name === name);
+};
+
+const getStartedTimeForCVDesiredVersion = (
+  cv: ClusterVersionKind,
+  desiredVersion: string,
+): string => {
+  const desiredHistory: UpdateHistory = cv?.status?.history?.find(
+    (update) => update.version === desiredVersion,
+  );
+  return desiredHistory?.startedTime;
+};
+
+const getUpdatingTimeForMCP = (machineConfigPool: MachineConfigPoolKind): string => {
+  const updatingCondition = machineConfigPool?.status?.conditions.find(
+    (condition) => condition.type === MachineConfigPoolConditionType.Updating,
+  );
+  return updatingCondition?.lastTransitionTime;
+};
+
+const getUpdatedOperatorCount = (
+  clusterOperators: ClusterOperator[],
+  desiredVersion: string,
+): number => {
+  return clusterOperators?.filter((operator) => {
+    const progressingCondition = operator?.status?.conditions.find(
+      (condition) => condition.type === 'Progressing',
```

Comment:
`Progressing` should have [nothing to do with whether a ClusterOperator is judged "upgraded" or not][1].

[1]: https://github.com/openshift/cluster-version-operator/blob/28e4400eeb9ded7e09ff684e75780599cb25ec2c/docs/user/reconciliation.md#clusteroperator

Reply:
Thanks, @wking.  After looking at this again, @spadgett and I agree and have removed the check, which results in operator progress bar behaving more like expected.